src/apis/PanResponder/index.js

```diff
@@ -348,18 +354,19 @@ var PanResponder = {
         // Guard against the dispatch of two touch moves when there are two
         // simultaneously changed touches.
         if (gestureState._accountsForMovesUpTo === touchHistory.mostRecentTimeStamp) {
+          console.log('bail out')
```

Comment:
Whoops, that accidentally slipped in :) I ran the linter and was expecting it to catch that case. I've removed it and updated the pull request :)